concepts/key-concepts.md

```diff
@@ -62,74 +66,79 @@ streams.
 
-* Append specific information to the Event like an IP address or metadata.
-* Select a specific piece of the Event content.
-* Drop Events that matches certain pattern.
+- Append specific information to the Event like an IP address or metadata.
+- Select a specific piece of the Event content.
+- Drop Events that match a certain pattern.
 
 ## Tag
 
-Every Event that gets into Fluent Bit gets assigned a Tag. This tag is an internal string that is used in a later stage by the Router to decide which Filter or Output phase it must go through.
+Every Event ingested by Fluent Bit is assigned a Tag. This tag is an internal string
+used in a later stage by the Router to decide which Filter or
+[Output](data-pipeline/output.md) phase it must go through.
 
-Most of the tags are assigned manually in the configuration. If a tag is not specified, Fluent Bit will assign the name of the Input plugin instance from where that Event was generated from.
+Most tags are assigned manually in the configuration. If a tag isn't specified,
+Fluent Bit assigns the name of the [Input](data-pipeline/input.md) plugin
+instance where that Event was generated from.
 
 {% hint style="info" %}
-The only input plugin that **does NOT** assign tags is [Forward](../pipeline/inputs/forward.md) input. This plugin speaks the Fluentd wire protocol called Forward where every Event already comes with a Tag associated. Fluent Bit will always use the incoming Tag set by the client.
+The [Forward](../pipeline/inputs/forward.md) input plugin doesn't assign tags. This
+plugin speaks the Fluentd wire protocol called Forward where every Event already
+comes with a Tag associated. Fluent Bit will always use the incoming Tag set by the
+client.
 {% endhint %}
 
-A Tagged record must always have a Matching rule. To learn more about Tags and Matches check the [Routing](data-pipeline/router.md) section.
+A tagged record must always have a Matching rule. To learn more about Tags and
+Matches, see [Routing](data-pipeline/router.md).
 
 ## Timestamp
 
-The Timestamp represents the _time_ when an Event was created. Every Event contains a Timestamp associated. The Timestamp is a numeric fractional integer in the format:
+The timestamp represents the time an Event was created. Every Event contains a
+timestamp associated. A timestamp always exists, and is set by the Input plugin or
+discovered through a data parsing process.
+
+The timestamp is a numeric fractional integer in the format:
 
 ```javascript
 SECONDS.NANOSECONDS
 ```
 
-### Seconds
-
-It is the number of seconds that have elapsed since the _Unix epoch._
-
-### Nanoseconds
+where:
 
-Fractional second or one thousand-millionth of a second.
-
-{% hint style="info" %}
-A timestamp always exists, either set by the Input plugin or discovered through a data parsing process.
-{% endhint %}
+- `_Seconds_` is the number of seconds that have elapsed since the Unix epoch.
+- `_Nanoseconds_` is a fractional second or one thousand-millionth of a second.
```

Comment:
should "seconds" and "nanoseconds" be capitalized here to match the example above? or is that not necessary

Reply:
Oh, yah they should be, I didn't fix them in editing.